src/commons_codec/transform/mongodb.py

```diff
@@ -163,7 +162,7 @@ def convert_epoch(value: t.Any) -> float:
             datetime = dateparser.parse(value)
         else:
             raise ValueError(f"Unable to convert datetime value: {value}")
-        return calendar.timegm(datetime.utctimetuple())
+        return datetime.timestamp()
```

Comment:
The previous instruction behaves better in all situations, as it produces the same results even when invoked on systems running different time zones.